.eslintrc.js

```diff
@@ -59,6 +59,11 @@ module.exports = {
     // So leave the rule, but add an exception.
     'no-return-assign': ['error', 'except-parens'],
 
+    // Same as Airbnb's except it allows default exports.
+    'no-restricted-exports': ['error', {
+      restrictedNamedExports: ['then'],
+    }],
```

Comment:
This is a new rule in `eslint-config-airbnb-base` ([changelog](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/CHANGELOG.md)), it disallows default exports, e.g. `export { default } from './module'` or `export { smth as default }`. 

https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/es6.js#L63-L70